apps/meteor/client/views/room/providers/RoomProvider.tsx

```diff
@@ -27,6 +29,21 @@ const RoomProvider = ({ rid, children }: RoomProviderProps): ReactElement => {
 
 	const roomQuery = useReactiveQuery(['rooms', rid], ({ rooms }) => rooms.findOne({ _id: rid }));
 
+	const subscribeToRoom = useStream('room-data');
+
+	const queryClient = useQueryClient();
+
+	// TODO: move this to omnichannel context only
+	useEffect(() => {
+		if (!isOmnichannelRoom(room)) {
```

Comment:
Just noticed after the last change, `room` doesn't exist anymore at this point, which will crash the app. I'll adjust the code.

Reply:
I've to cautiously use suggestions... I usually just make a draft of an idea, not really ensuring if the code work 🙈